atlasdb-tests-shared/src/main/java/com/palantir/atlasdb/transaction/impl/AbstractSerializableTransactionTest.java

```diff
@@ -696,6 +703,27 @@ public void testMultipleReadsToSameColumnRange() {
         t1.commit();
     }
 
+    protected LockResponse getRowLock(String rowName) {
+        LockDescriptor rowLockDescriptor = AtlasRowLockDescriptor.of(
+                TEST_TABLE.getQualifiedName(),
+                PtBytes.toBytes(rowName));
+        return lock(rowLockDescriptor);
+    }
+
+    protected LockResponse getCellLock(String rowName, String colName) {
+        LockDescriptor cellLockDescriptor = AtlasCellLockDescriptor.of(
+                TEST_TABLE.getQualifiedName(),
+                PtBytes.toBytes(rowName),
+                PtBytes.toBytes(colName));
+        return lock(cellLockDescriptor);
+    }
```

Comment:
nit: I'd prefer calling these `acquireRowLock` and `acquireCellLock` - given the return type it's clear they're trying to do work, but I don't normally expect a getter to have side effects

Reply:
thanks! Done 👍